Assets/Scripts/MIDI2Event/Midi2Event.cs

```diff
@@ -72,6 +72,24 @@ public void Update(double deltaTime)
             }
         }
 
+        //returns the current number of seconds per beat
+        public double SecPerBeat()
+        {
+            return _usPerQuarter * US_TO_S;
```

Comment:
Maybe have a comment somewhere in this file saying what "us" is (since I assume this is an abbreviation for some music technical term, but I could not find it online since I kept getting results assuming I meant the United States, and that could be helpful for someone like me who is less familiar)

Reply:
us is the standard abbreviation for microseconds when the mu character isn't available, can still add a comment to prevent confusion